third_party/java/arcs/build_defs/internal/kotlin_serviceloader_registry_template.tmpl

```diff
@@ -0,0 +1,2 @@
+{particles}
```

Comment:
If there's nothing else to add to this file, you can use `actions.write` instead of `actions.expand_template`.

Possibly there's a nice comment you'd want to add to this generated file, which might make it worth keeping the template.

Reply:
I didn't think the file format permitted comments, but apparently it does

https://docs.oracle.com/javase/tutorial/ext/basics/spi.html#packaging-the-dictionary-service-in-a-jar-file

Reply:
Changed to actions.write